## joint_names
joint_names: [arm_1_joint, arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint, arm_6_joint, arm_7_joint]

## Cartesian controller parameters
chain_base_link: arm_podest_link
chain_tip_link: arm_7_link
root_frame: world

# twist controller parameters
twist_controller:
  controller_interface: cob_twist_controller/ControllerInterfaceJointStates

# frame_tracker + interactive_marker
frame_tracker:
  target_frame: arm_7_target
  marker_scale: 0.3
  movable_trans: true
  movable_rot: true
  pid_trans_x: {p: 4.0, i: 0.0, d: 0.0, i_clamp: 0.0}
  pid_trans_y: {p: 4.0, i: 0.0, d: 0.0, i_clamp: 0.0}
  pid_trans_z: {p: 4.0, i: 0.0, d: 0.0, i_clamp: 0.0}
  pid_rot_x: {p: 4.0, i: 0.0, d: 0.0, i_clamp: 0.0}
  pid_rot_y: {p: 4.0, i: 0.0, d: 0.0, i_clamp: 0.0}
  pid_rot_z: {p: 4.0, i: 0.0, d: 0.0, i_clamp: 0.0}
